Update X11_forwarding.md for slurm commands

alexknigge opened the pull request:
Updated tutorial with up to date commands with slurm.

alexknigge committed:
Update X11_forwarding.md for slurm commands

gmfricke reviewed (changes requested):
@Graviton28 please update and test for Easley

Graviton28 committed:
Revise X11 forwarding instructions for clarity

Updated instructions for X11 forwarding on Easley, using macOS and Windows, clarified installation steps for XQuartz and MobaXterm, and improved overall readability.

Graviton28 committed:
Merge branch 'master' into alexknigge-patch-1

Graviton28 committed:
Update X11_forwarding.md
